assets/src/containers/FilesAccessed.js

```diff
@@ -14,21 +23,120 @@ const styles = theme => ({
   paper: {
     padding: theme.spacing.unit * 2,
     color: theme.palette.text.secondary
+  },
+  formController: {
+    display: "flex",
+    marginTop: theme.spacing.unit * 2,
+    alignItems: "center",
+    justifyContent: "center"
   }
 })
 
 function FilesAccessed (props) {
   const { classes, match } = props
-  const currentCourseId = match.params.courseId
-  const [loaded, assignmentData] = useFilesAccessedAssignmentData(currentCourseId)
+  const currentCourseId = match.params.courseId;
+  const [minMaxWeek, setMinMaxWeek] = useState([1, 17]);
+  const [curWeek, setCurWeek] = useState(7);
```

Comment:
The current week is hardcoded but must be coming from the `/info` endpoint call

Reply:
done